tests/test_relative_free_energy.py

```diff
@@ -69,8 +69,7 @@ def run_triple(mol_a, mol_b, core, forcefield, n_frames, protein_path, n_eq_step
 
     assert vacuum_res.overlap_summary_png is not None
     assert vacuum_res.overlap_detail_png is not None
-    assert abs(np.sum(vacuum_res.all_dGs)) < 10.0
-    assert np.linalg.norm(vacuum_res.all_dGs) < 10.0
+    assert np.linalg.norm(vacuum_res.all_errs) < 10.0
```

Comment:
this is considered a standard transformation (lambda windows are 0, 0.01, 0.02) I'd hope the error is a lot less than 10kJ/mol!

Reply:
Note that lambda schedule is unused in `vacuum_res` (22 windows between 0 and 1 are simulated) -- fixing.

Reply:
>this is considered a standard transformation (lambda windows are 0, 0.01, 0.02) I'd hope the error is a lot less than 10kJ/mol!

Error threshold tightened in https://github.com/proteneer/timemachine/pull/835/commits/6d70c744189c3691694badca6d564aa1b6f4e2bc